remote_attestation/rust/src/message.rs

```diff
@@ -14,6 +14,12 @@
 // limitations under the License.
 //
 
+// Messages used to implement the Enclave Key Exchange Protocol (EKEP).
+//
+// Unlike other RPC messages these are expressed as Rust structs and serialized
```

Comment:
nit: these are not RPC messages, so perhaps rather "Unlike RPC message, ...."

Reply:
Eg what makes the Attestation Report an RPC message that these messages do not have

Reply:
Just to clarify further: for me the property that makes something RPC is that it is the input parameter to a remote function or the result from the function execution. While you could argue the handshaker steps are also remote functions being executed, I see them as part of the initial setup, not part of the remote calculation that the client is interested in.

Reply:
I don't know if it's the actual definition of RPC, but in my mind RPC suggests a mechanism for invoking arbitrary procedures / functions remotely. e.g. in gRPC , the RPC part comes from the fact that a message has the name of the method to invoke, and the payload of the request. in capnproto instead of the name there is a number corresponding to the method, but conecptually similar. Here, it is just a fixed sequence of frames back and forth, there are no "procedures" being invoked.

Reply:
Fair, thanks for the context. That does mean we probably don't want to delineate format use based on what technically is an RPC message. Replaced RPC messages with "messages exchanged between client & server" here 😅